packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -32,7 +32,7 @@ import {
 export type CallControlsProps = {
   callInvitationURL?: string;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
-  options?: boolean | CallControlOptions;
+  options?: true | CallControlOptions;
```

Comment:
Now that you return `<></>` on line 139, this should continue to be `boolean`? And we should update `CallPage.tsx` and `LobbyPage.tsx` to always create and pass in `CallControlProps` (even when `CallControlOptions` is `false`?

Reply:
I'll change that back to boolean and investigate the other two pages 😊